README.md

```diff
@@ -1,22 +1,114 @@
-# Little Esty Shop
+# LittleEstyShop
 
-## Background and Description
+Welcome to LittleEstyShop! This project is a web application that serves as a platform for merchants to manage their items and invoices. It also provides admins with tools to monitor merchant performance and handle invoices. The application consumes the Unsplash API to display images, including the app logo, item images, merchant images, and random photos on the merchant's dashboard.
 
-"Little Esty Shop" is a group project that requires students to build a fictitious e-commerce platform where merchants and admins can manage inventory and fulfill customer invoices.
+## Work Completed
+The project implements various features for merchants and admins, as well as API consumption for images. Here is a summary of the work completed:
 
-## Learning Goals
-- Practice designing a normalized database schema and defining model relationships
-- Utilize advanced routing techniques including namespacing to organize and group like functionality together.
-- Utilize advanced active record techniques to perform complex database queries
-- Practice consuming a public API while utilizing POROs as a way to apply OOP principles to organize code
+## Merchant Dashboard
```

Comment:
Since these are all part of `Work Completed` I'd suggest changing the subheadings down one level